include/tuvx/linalg/linalg.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <iostream>
+#include <random>
+#include <vector>
+
+namespace tuvx {
+namespace linalg {
+
+// tridiag matrix
+template <typename T> struct trid_mat {
```

Comment:
```suggestion
template <typename T> struct TridiagonalMatrix {
```
We've been trying to follow the [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules) and using whole words and specific styles for `ClassNames`, `class_members_`, `FunctionsNames()`, `function_arguments`

Reply:
Got it. I will make those changes.